doc/design.md

```diff
@@ -491,6 +491,31 @@ Covers:
 
 Needs: impl, itest
 
+## Workarounds
+
+### Log Rotation Workaround Criteria
+`dsn~log-rotation-workaround-criteria~1`
+
+WorkaroundManager applies the `LogRotatinWorkaround` if the exasol version is 7.0.x or lower.
```

Comment:
Maybe you can describe here more general what the manager does. I guess it can apply different workarounds in the future? Probably depending on the version?

Reply:
Good point. Improved.